bokeh/core/property/regex.py

```diff
@@ -34,62 +32,12 @@
     'Regex',
 )
 
+deprecated("bokeh/core/property/regex is deprecated, use bokeh/core/property/string module instead")
```

Comment:
I would be better to use dots instead of slashes to indicate modules. Otherwise add `.py` at the end of those paths.

Reply:
Also, preferably use the three argument variant of `deprecated()` with `version=(2, 0, 1)`.